nexus/db-queries/src/db/datastore/switch_port.rs

```diff
@@ -339,6 +338,21 @@ impl DataStore {
                                 p.delay_open.into(),
                                 p.connect_retry.into(),
                                 p.keepalive.into(),
+                                p.remote_asn.map(|x| x.into()),
+                                p.min_ttl.map(|x| u32::from(x).into()),
+                                p.md5_auth_key.clone(),
+                                p.multi_exit_discriminator.map(|x| x.into()),
+                                p.local_pref.map(|x| x.into()),
+                                p.enforce_first_as,
+                                match &p.allowed_import {
+                                    ImportExportPolicy::NoFiltering => false,
+                                    _ => true,
+                                },
+                                match &p.allowed_export {
+                                    ImportExportPolicy::NoFiltering => false,
```

Comment:
This is terrifying -- can we instead (a) construct the struct directly, or (b) pass in `p` to the `new` constructor (what I would prefer)?

Reply:
Took the preferred suggestion